aggregator/config.go

```diff
@@ -113,6 +124,18 @@ type Config struct {
 
 	// Synchornizer config
 	Synchronizer syncronizerConfig.Config `mapstructure:"Synchronizer"`
+
+	// SettlementBackend configuration defines how a final ZKP should be settled. Directly to L1 or over the Beethoven service.
+	SettlementBackend SettlementBackend `mapstructure:"SettlementBackend"`
```

Comment:
Could we use an enum here?

Reply:
It is already an enum in a way:

```go
// SettlementBackend is the type of the settlement backend
type SettlementBackend string

const (
	// AggLayer settlement backend
	AggLayer SettlementBackend = "agglayer"

	// L1 settlement backend
	L1 SettlementBackend = "l1"
)
```

If you mean enum using `iota`, than, it wouldn't be a string representation. I went with a string because, that is how it was configured on the `cdk-validium-node`. I am down with using anything though,

Reply:
```suggestion
	SettlementBackend SettlementBackend `jsonschema:"enum=agglayer,enum=l1" mapstructure:"SettlementBackend"`
```

Reply:
Fix: https://github.com/0xPolygonHermez/zkevm-aggregator/pull/87/commits/a042f3fef1f30184729f5517abb7904c2fdd80ef